File: webapp-net/Search/DxaSearchException.cs
using System;
using Sdl.Web.Common;

namespace Sdl.Web.Modules.Search
{
    public class DxaSearchException : DxaException
    {
        public DxaSearchException(string message, Exception innerException = null)
                : base(message, innerException)
        {
        }
    }
}

File: webapp-java/dxa-module-search/src/main/java/com/sdl/dxa/modules/search/DxaSearchException.java
package com.sdl.dxa.modules.search;

import com.sdl.webapp.common.exceptions.DxaException;

public class DxaSearchException extends DxaException {

    public DxaSearchException(String message, Exception innerException) {
        super(message, innerException);
    }

    public DxaSearchException(String message) {
        super(message);
    }

    public DxaSearchException(Throwable cause) {
        super(cause);
    }
}
